docs-src/src/examples/DataGrid.js.example

```diff
@@ -1,136 +1,165 @@
 <Row>
   <Col>
-    <DataGrid>
+    <DataGrid className="rev-DataGrid--striped">
       <DataGrid.Header>
         <Row>
           <Col medium={2}>
-            <p>Column Header</p>
+            <span>Column Header</span>
           </Col>
           <Col medium={2}>
-            <p>Column Header</p>
+            <span>Column Header</span>
           </Col>
           <Col medium={2}>
-            <p>Column Header</p>
+            <span>Column Header</span>
           </Col>
           <Col medium={2}>
-            <p>Column Header</p>
+            <span>Column Header</span>
           </Col>
           <Col medium={2}>
-            <p>Column Header</p>
+            <span>Column Header</span>
           </Col>
           <Col medium={2}>
-            <p>Column Header</p>
+            <span>Column Header</span>
           </Col>
         </Row>
       </DataGrid.Header>
       <DataGrid.HeaderSmall>
         <Row>
           <Col medium={2}>
-            <p>Small DataGrid Header</p>
+            <span>Small DataGrid Header</span>
           </Col>
         </Row>
       </DataGrid.HeaderSmall>
       <DataGrid.Body>
-        <DataGrid.Row>
+        <Row>
           <Col medium={2}>
-            <p><span>Column Header:</span> DataGrid text</p>
+            <span className="rev-DataGrid-columnHeader--small">Column Header:</span> <span>DataGrid text</span>
```

Comment:
Thats a really long class name haha. Can we not make this work the way you did the tables `<Table.HeaderInline>` (except it would be `<DataGrid.HeaderInline>`)?

Reply:
jk saw this was done in #261